### PR TITLE
Add support for AddValidator/RemoveValidator

### PR DESCRIPTION
**Pull request by webmaster128**
Belongs to https://github.com/iov-one/iov-core/issues/1112

### CODE REVIEW

File: packages/iov-bns/src/types.ts [hunk 1]
```diff
@@ -25,6 +26,24 @@ export interface CashConfiguration {
 
 // Governance
 
+export interface ValidatorProperties {
+  readonly power: number;
+}
+
+export interface Validator extends ValidatorProperties {
+  readonly pubkey: PubkeyBundle;
+  readonly power: number;
```
Comment (willclarktech) [inline packages/iov-bns/src/types.ts]: Do you need to specify this here?

Comment (webmaster128) [reply packages/iov-bns/src/types.ts]: In `encodeValidators` we need to set `power` when creating a [ValidatorUpdate](https://htmlpreview.github.io/?https://github.com/iov-one/weave/blob/v0.18.0/docs/proto/index.html#weave.ValidatorUpdate). I assumed not every validator gets the same power (whatever that means in Tendermint land), so I made it configurable.

Comment (willclarktech) [reply packages/iov-bns/src/types.ts]: I just mean because `Validator` already extends `ValidatorProperties`.

Comment (webmaster128) [reply packages/iov-bns/src/types.ts]: Ah great catch. That's a bug

**Pull request merged**